packages/cli/src/SeedCacheFile/seedCacheFile.ts

```diff
@@ -45,17 +74,15 @@ export default async function seedCacheFile(): Promise<void> {
   if (fetchMethod === 'split') {
     daysPerRequest =
       parseInt(
-        await inputPrompt(
-          'How many days would you like to fetch per request? [1]',
-        ),
+        await new Promise((resolve) => {
+          process.stdin.once('data', (data) => {
+            resolve(data.toString().trim())
+          })
+          console.log('How many days would you like to fetch per request? [1]')
+        }),
```

Comment:
Is there a reason this is replacing the default prompt behavior for `daysPerRequest`?

Reply:
The replacement of inputPrompt method with a promise make the code more flexible and less dependent on a specific method for reading user input. 
However, I have to admit after a few nights of sleep, now it's than the new code does not validate the user input to ensure that it is a valid integer....

So both options are viable and it depends on how you would like to handle user input validation.

Which one do you "prefer" ?